# This represents the data coming from an IRLock sensor

Header header

float32 signature
float64 pos_x
float64 pos_y
float64 size_x
float64 size_y
